service/discovery/azure/azure.go

```diff
@@ -42,6 +42,12 @@ import (
 	"clouditor.io/clouditor/internal/util"
 )
 
+const (
+	StorageComponent = "storage"
+	ComputeComponent = "compute"
+	NetworkComponent = "network"
+)
```

Comment:
Maybe we move these one level up s.t. all discoveries use the same constants, e.g. also AWS. What do you think?

Reply:
You are absolutely right. I will change that.

Reply:
If we change this we run into 'import cycle' errors. For now, I'll leave it like that.